src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -226,6 +226,21 @@ public static String desanitizeFromHtml(String sanitizedString) {
     }
 
     /**
+     * Returns true if string contains no characters to be sanitized and contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String string) {
```

Comment:
I think I would need the method for future scripts as well. Can I create a new helper class in teammates/scripts and move it there instead?